### PR TITLE
ACM-2210 console api endpoint

### PR DESCRIPTION
**Pull request by jeswanke**

### CODE REVIEW

File: backend/src/routes/ansibletower.ts [hunk 1]
```diff
@@ -33,6 +36,11 @@ export function ansibleTower(req: Http2ServerRequest, res: Http2ServerResponse):
       return respondBadRequest(req, res)
     }
 
+    // whitelist of apis our ui calls
```
Comment (KevinFCormier) [inline backend/src/routes/ansibletower.ts:39]: Please use the term "allow list" here instead.

File: backend/src/routes/ansibletower.ts [hunk 2]
```diff
@@ -13,6 +13,9 @@ interface AnsibleCredential {
   token: string
 }
 
+// must match ansiblePaths in frontend/src/resources/utils/resource-request.ts
+const ansiblePaths = ['/api/v2/job_templates/', '/api/v2/workflow_job_templates/']
+
 export function ansibleTower(req: Http2ServerRequest, res: Http2ServerResponse): void {
   const token = getToken(req)
   if (!token) return unauthorized(req, res)
```
Comment (KevinFCormier) [inline backend/src/routes/ansibletower.ts:21]: Are we going to check if the token is valid? That was the complaint of the defect.

Comment (KevinFCormier) [reply backend/src/routes/ansibletower.ts:21]: And this was what I was getting at in terms of checking all our APIs to make sure we handle them consistently.

File: backend/src/routes/ansibletower.ts [hunk 3]
```diff
@@ -13,6 +13,9 @@ interface AnsibleCredential {
   token: string
 }
 
+// must match ansiblePaths in frontend/src/resources/utils/resource-request.ts
+const ansiblePaths = ['/api/v2/job_templates/', '/api/v2/workflow_job_templates/']
```
Comment (KevinFCormier) [inline backend/src/routes/ansibletower.ts:17]: Please be aware that once this PR is merged, backend changes will be required to access new Ansible Tower APIs @RobDolares. (Related to [ACM-3842](https://issues.redhat.com/browse/ACM-3842))

**Comment by jeswanke** [comment]
for new ansible api calls we just need to add to that list--just don't want anyone using our console-api to use the /screwyouup path

**Pull request merged**